android/src/main/java/cz/ackee/extensions/android/DrawableExtensions.kt

```diff
@@ -16,6 +16,7 @@ fun backgroundDrawable(
     isButton: Boolean = false,
     @ColorInt checkedColor: Int = color,
```

Comment:
what if we introduce some intermediate object for colors? this function is becoming really big with number of parameters

Reply:
I don't see any advantage in this since we still use named parameters. Imo it'll just introduce complexity.

Reply:
Did you use it in your projects?

Reply:
not yet but detekt default threshold for number of params is 6 and maybe its not a bad idea to keep that in mind. But since in this functions almost all parameters have default value it does not make sense to introduce it here

Reply:
I think so as wel... I've never had problems with it from the usage perspective, because I rarely use more than 3-4 params at once.